src/lib/storage/storage_manager.cpp

```diff
@@ -10,38 +10,56 @@
 namespace opossum {
 
 StorageManager& StorageManager::get() {
-  return *(new StorageManager());
-  // A really hacky fix to get the tests to run - replace this with your implementation
+  static StorageManager storage_manager = StorageManager();
+  return storage_manager;
 }
 
 void StorageManager::add_table(const std::string& name, std::shared_ptr<Table> table) {
-  // Implementation goes here
+  _table_names.emplace_back(name);
+  _tables.emplace_back(table);
 }
 
 void StorageManager::drop_table(const std::string& name) {
-  // Implementation goes here
+  size_t index = _get_index_for_table(name);
```

Comment:
```suggestion
  auto index = _get_index_for_table(name);
```
Due to consistency please decide on one variant (see l. 29).